src/libstd/lib.rs

```diff
@@ -425,46 +403,62 @@ pub use core::u16;
 pub use core::u32;
 #[stable(feature = "rust1", since = "1.0.0")]
 pub use core::u64;
+#[stable(feature = "rust1", since = "1.0.0")]
+pub use alloc::boxed;
```

Comment:
I think the ordering here affects the ordering in rustdoc, right? (just confirming you want to change that too)